src/settings.h

```diff
@@ -101,6 +103,30 @@ struct settings {
 
 extern struct settings settings;
 
+struct setting {
+      char *name; //config key
+      char *section; //config section
+      // TODO implement cmdopt
+      char *cmdopt; // cmdline option
```

Comment:
I agree. We should still provide some command line options though. Or do you think none of the settings need command line options? What does @tsipinakis think about it?
The command line options definitely need a refactor though, because now, there's no way to detect if an unknown option is passed.